### DISCUSSION TITLE
Join Strings through Rapids

### DISCUSSION THREAD

**Discussion by mtsol** [NVIDIA/spark-rapids#7795] [Q&A]
Hi, if I have two spark columns, how can I join those using rapids, and which function is available in cudf for that if I am trying to do that in evalueteColumnar function?

**Comment by jlowe** [comment]
Can you elaborate a bit on what you mean by "join"?  Given the reference to `evaluateColumnar` I assume this is for a simple UDF expression which must return one output row for every input row.  That implies "join" isn't like a SQL join.  Does "join" mean concatenating the strings together for each row like the [Spark string concatenation function, i.e.: `concat`](https://github.com/NVIDIA/spark-rapids/blob/branch-23.04/docs/configs.md#sql.expression.Concat)?  If that's the case, this is already implemented by the RAPIDS Accelerator.  See https://github.com/NVIDIA/spark-rapids/blob/branch-23.04/sql-plugin/src/main/scala/org/apache/spark/sql/rapids/collectionOperations.scala#L60-L66 for the source code.

If "join" means something else, it would be great to have a toy example showing the semantics of the operation required.
